### PR TITLE
framework/bluetooth: Implement bluetooth framework APIs

### PR DESCRIPTION
**Pull request by Wootak**

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [0318a9360a16aee55469ac41bdf8a9d032eae670] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0318a9360a16aee55469ac41bdf8a9d032eae670] - Code Rule Check OK.

File: framework/src/bluetooth/adaptation/bt-adatation-adapter.c [hunk 1]
```diff
@@ -44,12 +45,16 @@ static void __bt_enable_cb(int err)
 
 int bt_adapt_enable(void)
 {
+	int ret;
```
Comment (pillip8282) [inline framework/src/bluetooth/adaptation/bt-adatation-adapter.c]: it's not critical, svace checks initialization of variable. so how about initialize it?

**Comment by seinfra** [comment]
Target : [5fb7fbc13285757684694581f0a0551f05abc672] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [5fb7fbc13285757684694581f0a0551f05abc672] - Code Rule Check OK.

**Pull request merged**